src/go/rdctl/cmd/snapshotRestore.go

```diff
@@ -12,19 +12,21 @@ var snapshotRestoreCmd = &cobra.Command{
 	Use:   "restore <id>",
 	Short: "Restore a snapshot",
 	Args:  cobra.ExactArgs(1),
-	RunE: func(cmd *cobra.Command, args []string) error {
-		paths, err := paths.GetPaths()
-		if err != nil {
-			return fmt.Errorf("failed to get paths: %w", err)
-		}
-		manager := snapshot.NewManager(paths)
-		if err := manager.Restore(args[0]); err != nil {
-			return fmt.Errorf("failed to restore snapshot %q: %w", args[0], err)
-		}
-		return nil
-	},
+	RunE:  stopBackendCallFuncAndRestartBackend(restoreSnapshot),
 }
 
 func init() {
 	snapshotCmd.AddCommand(snapshotRestoreCmd)
 }
+
+func restoreSnapshot(cmd *cobra.Command, args []string) error {
```

Comment:
`cmd` could be `_` because it isn't used.

And `paths` should be `appPaths`, or something else.

Reply:
Regarding `cmd`, done.

Changed to importing `paths` package as `p`, as discussed.